packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -995,6 +1021,32 @@ export class GarbageCollector implements IGarbageCollector {
 		}
 	}
 
+	/**
+	 * Submit a GC op indicating that the Tombstone with the given path has been loaded.
+	 * Broadcasting this information in the op stream allows other clients to prevent
```

Comment:
I would not say "allows other clients to prevent this object from being deleted". Only summarizer client can do that right. Other clients will simply learn that this thing was "referenced via usage" (or something like that).